### PR TITLE
[collect_artifacts] Build MStore Build

### PR DESCRIPTION
**Pull request by anatoly-os**
Don't use it!

+ Fix workspaces read/write issue in the Store package environment

### CODE REVIEW

File: mscore/workspace.cpp [hunk 1]
```diff
@@ -1003,15 +1007,14 @@ QList<Workspace*>& Workspace::workspaces()
...
                               p = new Workspace;
-                              p->setPath(s + "/" + entry);
-                              p->setName(name);
-                              
-                              if (translate)
-                                    p->setTranslatableName(name);
-                              
-                              p->setReadOnly(!fi.isWritable());
-                              _workspaces.append(p);
-                              }
```
Comment (Jojo-Schmitz) [inline mscore/workspace.cpp]: This brace needs to stay. Or the opening one needs to go

File: mscore/workspace.cpp [hunk 2]
```diff
@@ -1003,15 +1007,14 @@ QList<Workspace*>& Workspace::workspaces()
                         
                         if (!p) {
```
Comment (Jojo-Schmitz) [inline mscore/workspace.cpp]: this brace needs to go, or the closing one needs to stay

File: mscore/workspace.cpp (hunk 1 above)
Comment (anatoly-os) [reply mscore/workspace.cpp]: Ah, merging empty strings is a nightmare! Thanks for catching it :)

**Comment by Jojo-Schmitz** [comment]
compile error: undefined reference to `qt_ntfs_permission_lookup'

**Comment by Jojo-Schmitz** [comment]
Travis CI / Linux build canceled, so no way to see whether that compilation issue got fixed there?
The AppVeyor / Windows build worked before anyway.

**Comment by anatoly-os** [comment]
The build is for Win packages only, so we don't care about other builds. Created a branch in the MU repo, btw.

**Comment by Jojo-Schmitz** [comment]
Would be better to not have to do this time and time again, i.e. somehow integrate those changes into master.

**Pull request closed without merging**